src/OpenTelemetry.Exporter.Console/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Bug fix, ConsoleExporter fails silently when exporting an `ActivityLink`
```

Comment:
nit: Reword this to something like:
Bug fix to prevent ConsoleExporter from failing when exporting an `ActivityLink` without tags.

Reply:
Not a blocker.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3984